include/envoy/server/factory_context.h

```diff
@@ -138,6 +139,11 @@ class ServerFactoryContext : public virtual CommonFactoryContext {
    * @return ServerLifecycleNotifier& the lifecycle notifier for the server.
    */
   virtual ServerLifecycleNotifier& lifecycleNotifier() PURE;
+
+  /**
+   * @return the flush interval of stats sinks.
```

Comment:
nit: return type

Reply:
will update to "@return std::chrono::milliseconds the flush interval of stats sinks."